src/main/java/com/abramchik/taskOnePatterns/behavioral/chainOfResponsibility/Notifier.java

```diff
@@ -0,0 +1,24 @@
+package com.abramchik.taskOnePatterns.behavioral.chainOfResponsibility;
+
+public abstract class Notifier {
+    private int priority;
+    private Notifier nextNotifier;
+
+    public Notifier(int priority) {
+        this.priority = priority;
+    }
+
+    public void setNextNotifier(Notifier nextNotifier) {
+        this.nextNotifier = nextNotifier;
+    }
```

Comment:
Use lombok for setters, getters, constructors etc

Reply:
done